src/protocol/templates/ping_doc.in

```diff
@@ -23,7 +23,7 @@
 |------|------|------------------|------------------------------------------------------------------|
 {% for sensor in sensors | sort %}
 {% set messages = sensors[sensor]['messages'] %}
-{% if sensor == 'Common' %}
```

Comment:
this should probably be part of the last commit where it is lowercased in the json.